src/olympia/scanners/admin.py

```diff
@@ -512,24 +513,29 @@ def handle_false_positive(self, request, pk, *args, **kwargs):
             'Scanner result {} has been marked as false positive.'.format(pk),
         )
 
-        title = 'False positive report for ScannerResult {}'.format(pk)
-        body = render_to_string(
-            'admin/false_positive_report.md', {'result': result, 'YARA': YARA}
-        )
-        labels = ','.join(
-            [
-                # Default label added to all issues
-                'false positive report'
-            ]
-            + ['rule: {}'.format(rule.name) for rule in result.matched_rules.all()]
-        )
+        if result.scanner == scanners.CUSTOMS:
+            title = 'False positive report for ScannerResult {}'.format(pk)
+            body = render_to_string(
+                'admin/false_positive_report.md', {'result': result, 'YARA': YARA}
```

Comment:
@wagnerand it's also probably safe to remove `, 'YARA': YARA`, it's not used in the template anymore